bazel/thirdparty/seastar.BUILD

```diff
@@ -674,6 +674,7 @@ cc_library(
         # needed in the testing library
         "include/seastar/testing/on_internal_error.hh",
     ],
+    copts = ["-U_FORTIFY_SOURCE"],
```

Comment:
is this one necessary? afaik we were thinking about its affects on longjmp, and that's only used in a .cc file in the main seastar library.

Reply:
it is used in the benchmarks, without this all benchmarks fail

Reply:
what does it affect in the benchmark? i mean, it's not totally important what exactly it affects, but the theory had been that it is only needed for `thread.cc`, which is not compiled as part of the testing library. therefore, it is affecting something else that we don't know about. given that seastar undefines fority source _for all of its dependencies_ then do we need to do the same thing?